packages/manager/apps/hycu/src/hooks/order/useOrderCatalogHYCU.ts

```diff
@@ -0,0 +1,21 @@
+import { useQuery } from '@tanstack/react-query';
+import { OvhSubsidiary } from '@ovh-ux/manager-react-components';
+import { ApiError } from '@ovh-ux/manager-core-api';
+import { getOrderCatalogHYCU } from '../../data/api/orderCatalogHYCU';
+import { HYCUCatalog } from '@/types/orderCatalogHYCU.type';
+
+export type OrderCatalogProps = {
+  ovhSubsidiary: OvhSubsidiary;
+};
+
+export const useOrderCatalogHYCU = (ovhSubsidiary: OvhSubsidiary) => {
+  const options: Record<string, unknown> = {
+    refetchOnWindowFocus: false,
+    keepPreviousData: true,
+  };
+  return useQuery<HYCUCatalog, ApiError>({
+    queryKey: ['order/catalog/public/licenseHycu', ovhSubsidiary],
+    queryFn: () => getOrderCatalogHYCU(ovhSubsidiary),
+    ...options,
+  });
+};
```

Comment:
```suggestion
export const useOrderCatalogHYCU = (ovhSubsidiary: OvhSubsidiary, Partial<DefinedInitialDataOptions<HYCUCatalog, ApiError>>) => {
  return useQuery({
    queryKey: ['order/catalog/public/licenseHycu', ovhSubsidiary],
    queryFn: () => getOrderCatalogHYCU(ovhSubsidiary),
    ...options,
  });
};
```

And now you can give option directly on the form component :   

```ts
useOrderCatalogHYCU(..., {
    refetchOnWindowFocus: false,
    keepPreviousData: true,
  });
  ```